config.json

```diff
@@ -6,7 +6,7 @@
     "pass": "pass"
   },
   "bitcoind": {
-    "url": "bitcoind01",
+    "url": "localhost",
```

Comment:
Is this change intended?

Reply:
yeap, though I'll remove this and do it in situ instad

Reply:
change removed